blueprints/in-repo-engine/routable-files/lib/__name__/addon/templates/application.hbs

```diff
@@ -1 +1 @@
-{{outlet}}
+{{outlet}}
```

Comment:
Meh, not a fan of the inconsistency, but if that's the default for apps, should be the same here.